Adyen/UI/List/ListItemView.swift

```diff
@@ -90,6 +89,37 @@ public final class ListItemView: UIView, AnyFormItemView {
         }
     }
     
+    private func updateTrailingView(for item: ListItem?) {
+        contentStackView.removeArrangedSubview(trailingView)
+        trailingView.removeFromSuperview()
+        
+        switch item?.trailingInfo {
+        case let .text(string):
+            let trailingTextLabel = UILabel()
+            trailingTextLabel.translatesAutoresizingMaskIntoConstraints = false
+            trailingTextLabel.text = string
+            trailingTextLabel.accessibilityIdentifier = item?.identifier.map {
+                ViewIdentifierBuilder.build(scopeInstance: $0, postfix: "trailingTextLabel")
+            }
+            trailingView = trailingTextLabel
+            trailingView.isHidden = string.isEmpty
+        case let .logos(urls, trailingText):
+            let trailingLogosView = SupportedPaymentMethodLogosView(
+                imageUrls: urls,
+                trailingText: trailingText
+            )
+            trailingLogosView.accessibilityIdentifier = item?.identifier.map {
+                ViewIdentifierBuilder.build(scopeInstance: $0, postfix: "trailingLogosView")
+            }
+            trailingView = trailingLogosView
```

Comment:
trailingView may be hidden from a previous case I think. We can either set it visible at the top of the function or set it here (and maybe even add tests to make sure?)

Reply:
I will write tests for it once it gets used by the PayByBank payment method - right now the new view is not in use yet 👌